### PR TITLE
Update score.py

### PR DESCRIPTION
**Pull request by jpe316**
Updating to not use getmodelpath

### CODE REVIEW

File: code/scoring/score.py [hunk 1]
```diff
@@ -33,8 +33,8 @@ def init():
     global model
 
     # load the model from file into a global object
-    model_path = Model.get_model_path(
-        model_name="sklearn_regression_model.pkl")
+    model_path = os.path.join(os.getenv('AZUREML_MODEL_DIR'),
```
Comment (eedorenko) [inline code/scoring/score.py]: This change causes the build fail. See the checks.

Comment (dtzar) [reply code/scoring/score.py]: Appears to be just linting issues

Comment (eedorenko) [reply code/scoring/score.py]: **os** is used, but not imported

**Comment by dtzar** [comment]
@jpe316 Was there a problem with using the current approach or improvement you hoped for via ENV var?
If we add the new ENV var, then we should also put it with a default value which works for training that I don't see in your PR.  Also you still need to fix the build error.

**Comment by jpe316** [comment]
> approach or improvement you hoped for via ENV var?
> If we add the new ENV var, then we should also put it with a default value which works for training that I don't see in your PR. Also you still need to fix the build error.

Yes, this means the model name is no longer hard-coded into the deployment. It also works across language boundaries. It also removes the dependency on the azure ML SDK just to look up a model name :)

**Comment by dtzar** [comment]
I believe with the code we now have here in [score.py](https://github.com/microsoft/MLOpsPython/blob/master/diabetes_regression/scoring/score.py#L39-L49) and [train.py](https://github.com/microsoft/MLOpsPython/blob/master/diabetes_regression/training/train.py#L111-L112) @jpe316 this satisfies the intent you had behind this PR.  There are also merge conflicts.  Ok to close?

**Comment by sudivate** [comment]
closing this pr as it is outdated from the current state of repo

**Pull request closed without merging**